src/client_side_request.cc

```diff
@@ -1136,7 +1136,7 @@ clientInterpretRequestHeaders(ClientHttpRequest * http)
 
     if (req_hdr->has(Http::HdrType::X_FORWARDED_FOR)) {
         String s = req_hdr->getList(Http::HdrType::X_FORWARDED_FOR);
-        fvdbCountForw(s.termedBuf());
+        fvdbCountXFF(StringToSBuf(s));
```

Comment:
Please use the standard header "Forwarded" as basis for naming not the deprecated XFF header.
```suggestion
        fvdbCountForwarded(StringToSBuf(s));
```

Reply:
Sure, but then shouldn't we counting Forwarded occurrences instead of XFF? Or even both?

Reply:
I thought about requesting count of both. But the PR for actually using Forwarded is not merged yet, and it makes a bit more sense for those user visible changes to this report be added with or after that PR.